store/migration/10001__init_schema.sql

```diff
@@ -1017,6 +1021,11 @@ UPDATE
     ON deployment_config FOR EACH ROW
 EXECUTE FUNCTION trigger_update_updated_ts();
 
+-- Define sheet enum types.
+CREATE TYPE sheet_visibility AS ENUM ('PRIVATE', 'PROJECT', 'PUBLIC');
+CREATE TYPE sheet_source AS ENUM ('BYTEBASE', 'VCS');
```

Comment:
VCS -> GITLAB_SELF_HOST

GitLab.com could be different, and GitHub would definitely be different